test/functional/test-performance.js

```diff
@@ -195,9 +195,9 @@ describe('performance', () => {
         expect(perf.events_.length).to.equal(2);
 
         return perf.coreServicesAvailable().then(() => {
-          expect(flushSpy.callCount).to.equal(2);
+          expect(flushSpy.callCount).to.equal(3);
```

Comment:
Hmm. I don't really understand these. Even worse, I don't see how we can test flush calls from that `whenDocumentComplete` block above since it's a completely separate microtask...